awx/settings/development.py

```diff
@@ -95,7 +95,6 @@
 include(optional('/etc/tower/conf.d/*.py'), scope=locals())
 
 BASE_VENV_PATH = "/var/lib/awx/venv/"
-ANSIBLE_VENV_PATH = os.path.join(BASE_VENV_PATH, "ansible")
```

Comment:
After searching around the current code base, I figured we need to bite the bullet here.

This location doesn't exist. Keeping around the setting can only cause trouble.